content/news/release-20181215-kknd.md

```diff
@@ -0,0 +1,28 @@
+While work continues on the next OpenRA playtest, we have collected a number of bugfixes that we wanted to fix before the holiday season. We are today pushing a bugfix release, [release-20181215](/download/), with the following improvements:
+
+* Fixed servers showing as "Waiting" in the server list after they have started.
+* Fixed a crash in the lobby if the admin changes map while a player is joining.
+* Fixed part of the lobby UI disappearing after transferring admin to another player.
+* Fixed players showing as "Anonymous" in the lobby after switching from another mod
+* Fixed a crash when using the /levelup cheat with units that are already veteran.
+* Fixed rare crashes when using Attack Move, and when ordering aircraft to land.
+* Fixed bugs with the Chronoshift killing Construction Yards and duplicating MCVs.
+* Fixed a glitch if the owner surrenders while a tech-structures is being captured.
+* Fixed a rare bug where units become uncontrollable after exiting a transport.
+* Fixed compatibility issues with certain older GPUs.
+* Removed tech-structure abilities from campaign-specific buildings in Red Alert.
+
+Head over to the [downloads page](/download/) to grab the new release, and spread the word!
+
+<hr />
+
+In other news, we are pleased to announce the first public alpha of the [Krush Kill 'n' Destroy remake](https://www.kknd-game.com/) project, which aims to rebuild and improve the first two KKnD games using the OpenRA game engine.
+
+[blurb from IceReaperrr: a couple of sentences about what KKnD is and the history/plan of/for the project]
```

Comment:
After the destruction of earths surface by the great nuclear war, those unlucky to survive mutated due to the high radiation. To make sure this never happens again, the so called evolved found their way back to their spiritual roots and relying on animals. Till 60 years later, the survivors which were hiding underground came back to the surface to claim it causing a war for the overworld.
The KKnD remake project started back in 2011 and is now fully open source along with its first public release. While still in early alpha stage the current release is focused around KKnD 1 and multiplayer gameplay. Further plans for the future include singleplayer gameplay as well as the sequel KKnD 2: Krossfire.
KKnD features unique mechanics such as an individual upgrade system and a parallel contruction system. Besides that it comes with a lot of small tweaks and enhancements over the original game to meet up modern standards.